Cargo.toml

```
@@ -2,6 +2,8 @@
 name = "detsys-ids-client"
 version = "0.1.0"
 edition = "2021"
+licenes = "Apache-2.0"
```

```suggestion
license = "Apache-2.0"
```

also, could we get the full text in a LICENSE file...? Just for avoidance of doubt..

---

:lol: